pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -43,7 +44,14 @@ def get_units(self, *filenames):
                 'store',
                 'store__translation_project__project',
                 'store__translation_project__language'
-            ).values(
+            )
+
+        if not self.include_disabled_projects:
```

Comment:
I'd suggest to name the attribute `.exclude_disabled_projects`, that way the condition becomes `if self.exclude_disabled_projects:` and the cognitive load for the reader is lower.